First_section.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" type="text/css" href="First_section.css" />
+  </head>
+
+  <body>
+    <div class="container">
+      <header class="toolbar">
+        <h1 id="Logo">My Logo</h1>
+        <img id="menu" src="union.png" />
+      </header>
+
+      <main class="content">
+        <span class="text_header">
+          <p1 id="welcome"> I am Christian Glad to see you! </p1>
+        </span>
+
+        <p id="description">
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+
+        <span class="social">
+          <h2>LET'S CONNECT</h2>
+        </span>
+        <section class="icons">
+          <img id="twitter" src="Vector.png" />
+          <img id="linkedin" src="Linkedin icon.png" />
+          <img id="vector1" src="Vector (1).png" />
+          <img id="vector2" src="Vector (3).png" />
+          <img id="vector3" src="Vector (2).png" />
+        </section>
```

Comment:
- [ ] I would recommend you use a ul tag and use li in it individually, inside the li, you can put the img tag inside a tag so that the social links can be accessed by adding href to the a tag.